src/app/shared/issue-tables/IssueList.ts

```diff
@@ -0,0 +1,75 @@
+import { DataSource } from '@angular/cdk/collections';
+import { MatPaginator } from '@angular/material/paginator';
+import { MatSort } from '@angular/material/sort';
+import { BehaviorSubject, merge, Observable, Subscription } from 'rxjs';
+import { Issue } from '../../core/models/issue.model';
+import { applyDropdownFilter, DEFAULT_DROPDOWN_FILTER, DropdownFilter } from './dropdownfilter';
+import { FilterableSource } from './filterableTypes';
+import { paginateData } from './issue-paginator';
+import { getSortedData } from './issue-sorter';
+import { TABLE_COLUMNS } from './issue-tables-columns';
+import { searchFilter } from './search-filter';
+
+/**
+ * Class similar to IssueDataTable but instead of listening to issueService,
+ * it will listen to an observable instead
+ */
```

Comment:
Do you mean that this class will listen for changes in a list containing any issues rather than the list of issues that `issueService` provides?